mitosheet/src/types.tsx

```diff
@@ -850,6 +851,14 @@ export interface UIState {
         // TODO: Move the other popups (loading, tour, fast forward) to use this infrastructure
         [PopupLocation.TopRight]: PopupInfo 
     }
+    dataRecon: { 
+        created_dataframe_names: string[],
```

Comment:
Intesting that we do all of these as strings. Is this a function what we return from the backend?

Reply:
Hmm tbh I'm not sure. I just reused the types that we already had.